browser/autoplay/autoplay_browsertest.cc

```diff
@@ -131,15 +122,15 @@ class AutoplayPermissionContextBrowserTest : public InProcessBrowserTest {
   void GotoAutoplayByMethod(bool muted) {
     bool clicked;
     if (muted)
-    ASSERT_TRUE(ExecuteScriptAndExtractBool(
-        contents(),
-        "window.domAutomationController.send(clickAutoplayByMethodMuted())",
-        &clicked));
+      ASSERT_TRUE(ExecuteScriptAndExtractBool(
```

Comment:
would be nice to occasionally migrate these to `EvalJs`

Reply:
addressed in https://github.com/brave/brave-core/pull/7343/commits/3139441b292b8b9809d51a15cd248155489203e2